orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/cdp-token.xml.j2

```diff
@@ -35,11 +35,11 @@
          <enabled>true</enabled>
          <param>
             <name>knox.acl.mode</name>
-            <value>AND</value>
+            <value>OR</value>
          </param>
          <param>
             <name>knoxtoken.acl</name>
-            <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
```

Comment:
The change looks good, I am just worried about restrictive permissions. We might want to test this in mow-dev. Would love to volunteer for some quick testing.

Reply:
@sidseth sorry for the delay in responding. I meant we are going from `*` user to a `gateway:autoscaleMachineUser` but this change is infact non-restrictive so there should not be any issues, i missed the "OR" part. 
But, yes the changes look good to me!